apis/vshn/v1/vshn_nextcloud.go

```diff
@@ -55,6 +55,15 @@ type VSHNNextcloudSpec struct {
 	WriteConnectionSecretToRef LocalObjectReference `json:"writeConnectionSecretToRef,omitempty"`
 }
 
+type VSHNNextcloudBackupSpec struct {
+	K8upBackupSpec `json:",inline"`
+
+	// +kubebuilder:default=false
+
+	// SkipMaintenance defines, if setting maintenance mode should be skipped during the backup. Defaults to false
```

Comment:
Please also add the implications of skipping the maintenance here.

Reply:
Done. Is the wording like that ok?